### PR TITLE
Feature/add submodule

### PR DESCRIPTION
**Pull request by quosi**
I added the data set as submodule to our git repository and updated the README file with project description and usefull references. Please, check (any team member can do this) and merge when approved. Thanks!

### CODE REVIEW

File: README.md
Comment (grafkevin) [inline README.md:3]: <b>ChestX</b> is a <b>TechLabs Berlin</b> project that aims to detect COVID-19 infection by processing chest X-ray images. To support medical decision making, the ChestX team wants to apply AI techniques for object detection & classification, as well as data science methods. Metadata provided alongside the X-ray images can be analysed to provide doctors with more meaningful diagnostic assistance.

File: README.md
Comment (grafkevin) [inline README.md:15]: Referencing the COVID-19 image data collection repository, some ML models were developed alongside the data collection process and constitute the main purpose of this research project.

**Review by grafkevin — CHANGES_REQUESTED** [review]
I corrected some typos in my comments. I don't know if that's the most elegant way to review documents though *hihi*

File: README.md
Comment (quosi) [reply README.md:3]: Thanks! That's geat. I corrected the two typos and updated the file. Would merge this now after a second review to approve the changes.

File: README.md
Comment (quosi) [reply README.md:15]: corrected the typos

**Pull request merged**
